api/src/main/resources/plugins/flink.py

```diff
@@ -0,0 +1,176 @@
+"""
```

Comment:
General comment - there is a lot of common code here with the spark streaming plugin. Can you factor this into a base class?

Reply:
Created new derive class from base_creator with common methods of flink.py and sparkStreaming.py plugins and used the same for flink.py and sparkStreaming.py .